### ISSUE TITLE
Enhancement(UX): Add traffic light buttons to Hook fail dialog

### ISSUE THREAD

**Issue by gihrig** [git-up/GitUp#251]
GitUp 1.0.6 (1020)
Mac OS 10.11.6
Screen res 1280x800, 20" wide aspect monitor

When a pre-commit hook fails with more data than will fit on screen the 'OK' button is not visible and cannot be moved on screen. This this leaves no visible way to dismiss the dialog and force quit seems to be required.

The fact is, the dialog can be dismissed by pressing enter, as the 'OK' button is the default control.

However, if the dialog had 'traffic light' buttons at the top, there would always be visible means to close the dialog.

In this screenshot there are only three staged files with lint errors, yet the 'OK' button is pushed off-screen.
![image](https://cloud.githubusercontent.com/assets/1481063/21212545/935cde94-c242-11e6-8529-a2f6929aa72a.png)

**Comment by swisspol** [comment]
This is actually a system modal dialog IIRC, so I don't think you can add the traffic lights. However, the text should be truncated.

**Comment by gihrig** [comment]
Understood. Thanks for the fast reply ✨

**Comment by capnslipp** [comment]
I agree with @swisspol.  Truncating the text would do the job, or (as I believe I've seen elsewhere) text in a scroll view.
